codes.md

```diff
@@ -117,4 +117,7 @@ A list of every Checka11y.css error & warning code with details on what the issu
   The highlighted element has been detected to contain a `title` attribute. There many [accessibility concerns with the `title` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#accessibility_concerns). Consider using an [alternative method to using the `title` attribute](https://inclusive-components.design/tooltips-toggletips).
 
 - ### W0009
-  The highlighted element is a focusable element that is nested within another element with `aria-hidden="true"`. This means the focusable element is inaccessible to assistive technologies. Either remove the `aria-hidden="true"`, or restructure the HTML so that the focusable element is not nested within the element with `aria-hidden="true"`. [Read more about this here](https://web.dev/aria-hidden-focus).
\ No newline at end of file
+  The highlighted element is a focusable element that is nested within another element with `aria-hidden="true"`. This means the focusable element is inaccessible to assistive technologies. Either remove the `aria-hidden="true"`, or restructure the HTML so that the focusable element is not nested within the element with `aria-hidden="true"`. [Read more about this here](https://web.dev/aria-hidden-focus).
+
+- ### W0010

```

Comment:
Really clear description. Just one minor change needed to the below sentence

"Those anchor elements ~with~ are often used to trigger a click event on the page."

Reply:
Yep, sure. Edited this description several times, needed to reread it :smile: